core/tests/protractor_utils/ExplorationEditorStatsTab.js

```diff
@@ -30,8 +31,17 @@ var ExplorationEditorStatsTab = function() {
     return numPassersby.getText();
   };
 
+  var _getInitIssueText = function() {
+    return firstIssueElement.getText();
+  };
+
   this.expectNumPassersbyToBe = function(numPassersby) {
     expect(_getNumPassersby()).toMatch(numPassersby);
+    firstIssueElement.click();
```

Comment:
Why is this part of `expectNumPassersbyToBe`? Shouldn't this be in its own function?

Reply:
Done.